src/components/Navigation.js

```diff
@@ -3,14 +3,17 @@ import { Link } from 'react-router-dom';
 
 export default function AppNav() {
   return (
-    <nav>
-      <div className="NavBar">
-        <h2>Bookstore CMS</h2>
-        <ul className="navSection">
+    <nav className="navContainer">
+      <div className="navItems">
+        <h1 id="navTitle">Bookstore CMS</h1>
+        <ul className="navPages">
           <li><Link to="/">Books</Link></li>
           <li><Link to="/categories">Categories</Link></li>
         </ul>
       </div>
+      <div className="icon">
+        <img src="https://img.icons8.com/external-anggara-outline-color-anggara-putra/32/000000/external-avatar-interface-anggara-outline-color-anggara-putra.png" alt="profile" />
```

Comment:
Your app is not performing as expected. It keeps refreshing every second. Please see the video attached for the behavior. Kindly fix it.

https://www.loom.com/share/7ac6af2d4150452f9a756f3f04c5995d